test/core/AnimatedSprite.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+describe('PIXI.extras.AnimatedSprite', function ()
+{
+    describe('instance', function ()
+    {
+        beforeEach(function ()
+        {
+            this.textures = [PIXI.Texture.EMPTY];
+        });
+
+        afterEach(function ()
+        {
+            expect(this.sprite._textures).to.be.eql(this.textures);
+            expect(this.sprite._durations).to.be.null;
+            expect(this.sprite.animationSpeed).to.be.equal(1);
+            expect(this.sprite.loop).to.be.true;
+            expect(this.sprite.onComplete).to.be.null;
+            expect(this.sprite.onFrameChange).to.be.null;
+            expect(this.sprite.onLoop).to.be.null;
+            expect(this.sprite._currentTime).to.be.equal(0);
+            expect(this.sprite.playing).to.be.false;
+
+            this.sprite.destroy();
```

Comment:
Make sure you set `this.sprite = null` after the destroy

Reply:
Btw we dont need destroy() there, because it really doesnt appear in caches and has no generated textures.

Reply:
anyway, just add "this.sprite=null" and it'll be fine.

Reply:
@ivanpopelyshev maybe stay `destroy()` anycase for reliability? if in  future it will be meaningful we will need to update tests.

Reply:
Leave destroy, it doesn’t hurt

Reply:
okey